blocks/bricks/conv.py

```diff
@@ -335,9 +335,15 @@ def get_dim(self, name):
         if name == 'input_':
             return self.input_dim
         if name == 'output':
-            return tuple(Pool.out_shape(
-                self.input_dim, self.pooling_size, st=self.step,
-                ignore_border=self.ignore_border, padding=self.padding))
+            if None in self.input_dim[-2:]:
```

Comment:
Hmm. Is having only one None in the input_dim a supported case? Taking a quick glance through the code, it looks like the check is always to see if it is equal to `(None, None)`. I can definitely change this code to reflect that. But it looks like if, say, the `Convolutional` class is given input dimensions where exactly one dimension is `None`, it'd break, since it wouldn't pass through the `(None, None)` check, and then would have some operations done on it that wouldn't work with a `None` value. So it seems unlikely that the `Pooling` class would ever be given such an input, but if that case were to be supported, it'd require similar changes to other classes as well, such as `Convolution`.